.nycrc

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
I looked what was this file online and I found this:

New York City Regional Center :: NYCRC 😄